app/src/main/java/com/imashnake/animite/dev/ext/SchemeExt.kt

```diff
@@ -0,0 +1,37 @@
+package com.imashnake.animite.dev.ext
+
+import androidx.core.graphics.ColorUtils
+import scheme.Scheme
+
+fun Scheme.pastelize(backgroundToPrimary: Float) =
+    Scheme(
```

Comment:
No `this.copy()`?

Reply:
It's because it's a java class or something.